nixos/modules/services/web-apps/peertube.nix

```diff
@@ -602,25 +599,23 @@ in {
             proxy_set_header Connection                 'upgrade';
 
             proxy_http_version                          1.1;
-          '';
+          '' + nginxCommonHeaders;
         };
 
         # Bypass PeerTube for performance reasons.
         locations."~ ^/client/(assets/images/(icons/icon-36x36\.png|icons/icon-48x48\.png|icons/icon-72x72\.png|icons/icon-96x96\.png|icons/icon-144x144\.png|icons/icon-192x192\.png|icons/icon-512x512\.png|logo\.svg|favicon\.png|default-playlist\.jpg|default-avatar-account\.png|default-avatar-account-48x48\.png|default-avatar-video-channel\.png|default-avatar-video-channel-48x48\.png))$" = {
           tryFiles = "/client-overrides/$1 /client/$1 $1";
           priority = 1310;
+
+          extraConfig = nginxCommonHeaders;
```

Comment:
If the nginx maintainers wouldn't be against it, we could simplify the code a lot with an option like https://github.com/SuperSandro2000/nixos-modules/blob/master/modules/nginx.nix#L63 :)

Reply:
Does this option apply to all sites, or is it configured individually for each site?

Reply:
That would apply to all sites but I think we could also add some logic to only inject it for one or some sites.